packages/paste-website/src/components/Footer.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Link} from 'gatsby';
+
+import {Anchor} from '@twilio-paste/anchor';
+import {TWILIO_BLUE} from '../constants';
+import TwilioIcon from './svg/twilio-icon';
+
+const StyledFooter = styled.footer`
+  margin-top: 96px;
+  padding: 120px ${themeGet('space.space200')} 120px;
+  background-color: ${themeGet('colors.colorGray10')};
+`;
+
+const StyledFooterGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column-gap: ${themeGet('space.space70')};
+`;
+
+const Styled2Column = styled.div`
+  grid-column: span 2;
+`;
+
+const StyledFooterList = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style: none;
+`;
+
+const StyledFooterLink = styled(Link)`
+  color: ${themeGet('colors.colorGray80')};
+  text-decoration: none;
+
+  &:hover {
+    color: ${themeGet('textColors.colorText')};
+  }
+`;
+
+const Styled4Column = styled.div`
+  grid-column: 9 / span 4;
+`;
+
+const StyledLinkList = styled.p`
+  a {
```

Comment:
cc @SiTaggart this kind of makes me think all components should accept margin tokens.  Thoughts?